packages/target-decisioning-engine/test/decisioning.spec.js

```diff
@@ -89,15 +89,33 @@ describe("decisioning engine", () => {
 
       fetch.mockResponses(...mockResponses);
 
+      function executeTelementries(request, entries) {
+        return {
+          ...request,
+          telemetry: {
+            entries
+          }
+        };
+      }
+
+      const telemetryProvider = TelemetryProvider(
+        executeTelementries,
+        conf.telemetryEnabled
+      );
+
       decisioning = await TargetDecisioningEngine({
         ...conf,
         sendNotificationFunc,
-        artifactFormat: ARTIFACT_FORMAT_JSON // setting this tells the artifactProvider deobfuscation is not needed
+        artifactFormat: ARTIFACT_FORMAT_JSON, // setting this tells the artifactProvider deobfuscation is not needed
+        telemetryProvider
       });
 
       expect(decisioning.getRawArtifact()).toEqual(artifact);
 
-      const result = await decisioning.getOffers(input);
+      const result = await decisioning.getOffers({
+        ...input
```

Comment:
Was there a particular reason you had to copy `input` into a new obj here?  Just curious

Reply:
ah, side effect to how I used to have telemetryProvider passed in the same object. Decided that implementation sucked and switched it but forgot to fix this